c/zss.c

```diff
@@ -1123,7 +1123,10 @@ static void initializePluginIDHashTable(HttpServer *server) {
 #define ZSS_STATUS_ERROR  8
 
 int main(int argc, char **argv){
-
+  if (argc == 1) { 
```

Comment:
Why do we need an additional check here? Isn't line 1139 doing the same?

Reply:
Please see my previous comment.

Reply:
yes it should be